README.md

```diff
@@ -19,4 +19,4 @@ Implement the ability to reorder the goods with the given buttons.
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_list-of-goods-js/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://KateLis84.github.io/react_list-of-goods-js/) and add it to the PR description.
```

Comment:
Add this demo link to the PR description 
<img width="971" alt="image" src="https://github.com/user-attachments/assets/21631a4a-d523-4753-8bbe-d06d0de9b42c" />